### DISCUSSION TITLE
Why is the downloaded picture changed to .avif format, I can't get the picture information

### DISCUSSION THREAD

**Discussion by mimiscript** [civitai/civitai#425] [Q&A]
Use Chrome,please help me,thx

**Comment by steelywing** [comment]
Can we get back the `.png` format? I used to download the image I like and then check the prompt later.

I know `.avif` file will be smaller, but can it preserve the prompt info? if not, can we use `.jpeg` and keep the prompt for saving storage space.

**Comment by JustMaier** [comment]
To those stumbling across this. We addressed this by adding an option to receive optimized image formats (avif, webp), or unoptimized formats (jpeg, png) to the browsing settings in your user account:
![image](https://user-images.githubusercontent.com/607609/231455519-54f5dd01-ba19-4235-830f-bc82cadfe869.png)

**Reply by diegocr** [reply]
Hello, are there any plans to support reading generation data from WebP files? Thanks!
